pom.xml

```diff
@@ -684,6 +672,7 @@
                             <usePhrasedTestCaseMethodName>true</usePhrasedTestCaseMethodName>
                         </statelessTestsetReporter>
                         <argLine>
+                            -Dio.netty.allocator.type=pooled
```

Comment:
What's the rationale for setting this in the tests?  Should we be setting this in the kroxylicious-start.sh?